### PR TITLE
Add quickstart guide for Node

### PR DESCRIPTION
**Pull request by murgatroid99**

### CODE REVIEW

**Comment by LisaFC** [comment]
LGTM. Kailash?

File: docs/quickstarts/node-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+(Don't forget to save the file!)
+
+## Update and run the appolication
```
Comment (hsaliak) [inline docs/quickstarts/node-quickstart.md]: Typo here -- s/appolication/application

Comment (murgatroid99) [inline docs/quickstarts/node-quickstart.md]: Fixed

**Comment by hsaliak** [comment]
LGTM

**Pull request merged**